src/Components/Person.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const getSexOfPartner = () => (sex === 'f' ? 'husband' : 'wife');
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">
+        {age && `I am ${age}`}
+      </p>
```

Comment:
you should use conditional rendering for the whole `p` element, it's no need to have an empty `p`